viral/basic_plots.py

```
@@ -0,0 +1,766 @@
+        for i, stage in enumerate(stages):
+            p_text = f"P = {round(p_values[f"{state}_{stage}"].values[0], 2)}"
+            axes[idx].text(i, text_y, p_text, ha="center", va="top")
+
+    plt.tight_layout(rect=[0, 0, 1, 0.95])
+    plt.savefig(
+        SERVER_PATH
+        / "viral_plots"
+        / "firing_rates"
+        / f"comparison_firing_rates_rewarded_{rewarded}.png"
+    )
+    1 / 0
```

Ruff (0.14.4)

Found useless expression. Either assign it to a variable or remove it.

(B018)